internal/validation.go

```diff
@@ -1,13 +1,23 @@
 package internal
 
-import "os"
+import (
+	"os"
 
-var TMDB_CLI_BEARER_TOKEN string = os.Getenv("TMDB_CLI_BEARER_TOKEN")
+	"github.com/joho/godotenv"
+)
 
 func ValidateBearerToken() string {
-	if TMDB_CLI_BEARER_TOKEN == "" {
+
+	err := godotenv.Load()
+	if err != nil {
```

Comment:
This is a problem because what happens when the .env file is not created? Please check if you can just load the envs from .env file without check errors 🙏